Project-Aurora/Project-Aurora/Profiles/RocketLeague/GSI/Nodes/Player.cs

```diff
@@ -1,48 +1,27 @@
 ﻿namespace Aurora.Profiles.RocketLeague.GSI.Nodes
 {
-    /// <summary>
-    /// Enum list for each player team
-    /// </summary>
-    public enum PlayerTeam
-    {
-        /// <summary>
-        /// Undefined
-        /// </summary>
-        Undefined = -1,
-
-        /// <summary>
-        /// Spectator
-        /// </summary>
-        Spectator = 400021321, //4294967295
-
-        /// <summary>
-        /// Blue Team
-        /// </summary>
-        Blue = 0,
-
-        /// <summary>
-        /// Orange Team
-        /// </summary>
-        Orange = 1
-    }
-
     /// <summary>
     /// Class representing player information
     /// </summary>
     public class Player_RocketLeague : Node<Player_RocketLeague>
     {
-        /// <summary>
-        /// Player's boost amount [0.0f, 1.0f]
-        /// </summary>
-        public int BoostAmount = 0;
-
-        /// <summary>
-        /// Player's current team
-        /// </summary>
-        public PlayerTeam Team = PlayerTeam.Undefined;
+        public int Team = -1;
```

Comment:
What is the meaning of this value? Does it mean the index of the player's team? How would someone get the gsi variables for their team?

Also, could do with some documentation on the variables in here